deeplearning4j-modelimport/src/main/java/org/deeplearning4j/nn/modelimport/keras/layers/embeddings/KerasEmbedding.java

```diff
@@ -66,12 +76,10 @@ public KerasEmbedding(Map<String, Object> layerConfig, boolean enforceTrainingCo
         this.inputShape[0] = inputShapeOld[0];
         this.inputShape[1] = inputDim;
 
-        boolean hasZeroMasking = KerasLayerUtils.getZeroMaskingFromConfig(layerConfig, conf);
+        this.hasZeroMasking = KerasLayerUtils.getZeroMaskingFromConfig(layerConfig, conf);
         if (hasZeroMasking)
-            log.warn("Masking in keras and DL4J work differently. We do not support mask_zero flag" +
-                    "on Embedding layers. If you want to have this behaviour for your imported model" +
-                    "in DL4J, apply masking as a pre-processing step to your input." +
-                    "See https://deeplearning4j.org/usingrnns#masking for more on this.");
+            log.warn("Zero Masking for the Embedding layer only works with unidirectional LSTM for now."
+                    + "For Bidirectional layers, you can set the mask yourself.");
```

Comment:
Could we maybe put the last two rows of the old docs back in? Please be a little more explicit what you mean by setting it yourself.